lib/stubzzz.js

```diff
@@ -0,0 +1,23 @@
+const prefilter = (bib, itemOrHolding) => {
+  const filter = async (records) => {
+    return await Promise.all(records.map((records) => Promise.resolve({ ...records, bibIds: [bib.id], [itemOrHolding]: { id: '123' } })))
+  }
+  return filter
+}
+
+const asyncNoop = async (x) => Promise.resolve(x)
+class EsBib {
+  constructor (bib) {
+    Object.keys(bib).forEach((key) => {
+      this[key] = bib[key]
+    })
```

Comment:
This is just a stub, but it looks like this pattern is messing with the `toJson` process run on this stubbed data later. I see errors when running this via `sam`. The actual implementation of `EsBib` should store the SierraBib model as an instance variable instead of cloning all of `bib`'s properties into itself. e.g.
```
class EsBib {
  constructor (bib) {
    this.bib = bib 
  }
  ...
}
```